diff --git a/vendor/magento/module-catalog/view/frontend/templates/product/list/toolbar.phtml
index 3ee2rd..8349152 111645
--- a/vendor/magento/module-catalog/view/frontend/templates/product/list/toolbar.phtml
+++ b/vendor/magento/module-catalog/view/frontend/templates/product/list/toolbar.phtml
@@ -3,6 +3,9 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+ /*
+ * Modified:by Emmpressit 01/07/22
+ */
 ?>
 <?php
 /**
@@ -16,17 +19,8 @@
         $widgetOptions = $this->helper(\Magento\Framework\Json\Helper\Data::class)->jsonEncode($widget['productListToolbarForm']);
     ?>
     <div class="toolbar toolbar-products" data-mage-init='{"productListToolbarForm":<?= /* @noEscape */ $widgetOptions ?>}'>
-        <?php if ($block->getIsBottom()): ?>
-            <?= $block->getPagerHtml() ?>
-            <?= $block->fetchView($block->getTemplateFile('Magento_Catalog::product/list/toolbar/limiter.phtml')) ?>
-        <?php else: ?>
-            <?php if ($block->isExpanded()): ?>
-                <?= $block->fetchView($block->getTemplateFile('Magento_Catalog::product/list/toolbar/viewmode.phtml')) ?>
-            <?php endif ?>
-            <?= $block->fetchView($block->getTemplateFile('Magento_Catalog::product/list/toolbar/amount.phtml')) ?>
-            <?php if ($block->isExpanded()): ?>
-                <?= $block->fetchView($block->getTemplateFile('Magento_Catalog::product/list/toolbar/sorter.phtml')) ?>
-            <?php endif ?>
-        <?php endif ?>
+        <?= $block->fetchView($block->getTemplateFile('Magento_Catalog::product/list/toolbar/sorter.phtml')) ?>
+        <?= $block->fetchView($block->getTemplateFile('Magento_Catalog::product/list/toolbar/limiter.phtml')) ?>            
+        <?= $block->getPagerHtml() ?>  
     </div>
 <?php endif ?>
